### PR TITLE
feat(user-keys):  show and copy users apiKey and secretKey

### PR DESCRIPTION
**Pull request by ksendart**
#745

### CODE REVIEW

File: src/app/shared/components/parameters-pair/parameters-pair.component.html [hunk 1]
```diff
@@ -17,10 +17,31 @@
...
       ngxClipboard
       class="value"
       [innerHTML]="value"
       [matTooltip]="value"
     ></div>
+    <cs-input-group *ngIf="isCopy" class="value">
+      <mat-form-field class="full-width-input" cs-input-group-textbox>
+        <input matInput [value]="value" readonly>
+      </mat-form-field>
+      <button
```
Comment (qqcky) [inline src/app/shared/components/parameters-pair/parameters-pair.component.html]: Looks like the button does not fit if Russian language is used and the label is wide

File: src/app/shared/components/parameters-pair/parameters-pair.component.ts [hunk 2]
```diff
@@ -14,5 +10,17 @@ export class ParametersPairComponent {
   @Input() public name: string;
   @Input() public value: string;
   @Input() public isEdit: boolean = false;
+  @Input() public isCopy: boolean = false;
```
Comment (qqcky) [inline src/app/shared/components/parameters-pair/parameters-pair.component.ts]: please, rename isCopy to something like canBeCopied

File: src/app/shared/services/user.service.ts [hunk 3]
```diff
@@ -16,9 +16,9 @@ const DEFAULT_SESSION_TIMEOUT = 30;
 @Injectable()
 @BackendResource({
   entity: 'User',
-  entityModel: User
+  entityModel: AccountUser
 })
-export class UserService extends BaseBackendService<User> {
+export class UserService extends BaseBackendService<AccountUser> {
```
Comment (qqcky) [inline src/app/shared/services/user.service.ts]: Can we merge UserService and AccountUserService?

File: src/app/shared/components/parameters-pair/parameters-pair.component.ts [hunk 4]
```diff
@@ -13,6 +9,18 @@ import {
 export class ParametersPairComponent {
   @Input() public name: string;
   @Input() public value: string;
-  @Input() public isEdit: boolean = false;
+  @Input() public canBeEdit: boolean = false;
```
Comment (zolotyx) [inline src/app/shared/components/parameters-pair/parameters-pair.component.ts]: strange naming, can we use canBeCopied or allowCopy

**Pull request merged**